patches/chrome-installer-util-google_update_settings.cc.patch

```diff
@@ -14,23 +14,48 @@ index 441ed6d50d5cb4edcd665665fca7887f034ff6a0..709c7e5e24518902ae86ab1ef819dd9d
  const wchar_t GoogleUpdateSettings::kUpdatePolicyValue[] = L"UpdateDefault";
  const wchar_t GoogleUpdateSettings::kDownloadPreferencePolicyValue[] =
      L"DownloadPreference";
-@@ -41,6 +45,11 @@ const wchar_t GoogleUpdateSettings::kCheckPeriodOverrideMinutes[] =
- const int GoogleUpdateSettings::kCheckPeriodOverrideMinutesMax =
-     60 * 24 * 7 * 6;
+@@ -43,7 +47,7 @@ const int GoogleUpdateSettings::kCheckPeriodOverrideMinutesMax =
  
-+
-+#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
-+#define GOOGLE_CHROME_BUILD
-+#endif
-+
  const GoogleUpdateSettings::UpdatePolicy
      GoogleUpdateSettings::kDefaultUpdatePolicy =
- #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
-@@ -836,3 +845,7 @@ bool GoogleUpdateSettings::ReadExperimentLabels(
- 
-   return result == ERROR_SUCCESS;
+-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
++#if BUILDFLAG(GOOGLE_CHROME_BRANDING) || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD))  // NOLINT
```

Comment:
we do not want to enable google updates, I don't understand this change

Reply:
These aren't functional changes, we had it enabled before. 
The only difference is that before C78 this was under 
`#ifdef GOOGLE_CHROME_BUILD` 
and we used to define it just for the file (either via patch or via chromium_src).
Now, in C78 they are replacing `#ifdef GOOGLE_CHROME_BUILD` with
 `#if BUILDFLAG(GOOGLE_CHROME_BRANDING)`
If we wanted to make that enabled for the file, we'd have to expand their macros and do something like
`#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
#define BUILDFLAG_INTERNAL_GOOGLE_CHROME_BRANDING 1
#endif
`
like we did before.
In other places we do patch with the || (defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)) and that gets us away from dependency on their future macro changes.

Reply:
fyi, we use open source `omaha` for windows update.
omaha is opensource version of `google update`.
And we reuse most of `google update` integration logic in chromium.